src/clojush/args.clj

```diff
@@ -177,6 +177,10 @@
           :silent-instruction-probability 0.2 
           ;; If :silent is used as an epigenetic-marker, this is the probability of random 
           ;; instructions having :silent be true.
+
+          :track-instruction-maps false
```

Comment:
This seems a non-obvious name to me. I don't necessarily object to it, it just wasn't obvious to me that it meant what the comment said it meant. Thoughts?